WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController/NotificationsViewController.swift

```diff
@@ -219,12 +219,13 @@ class NotificationsViewController: UIViewController, UITableViewDataSource, UITa
             return
         }
 
-        if shouldShowPrimeForPush {
-            setupNotificationPrompt()
+        if !UITestConfigurator.isEnabled(.disablePrompts) {
```

Comment:
Not a comment on the code, but I just think this check statement (if 'disablePrompts' is not disabled) is pretty funny 😄

Reply:
Yeah, it's not great because of the double negative. It's actually triple negative: "not" enabled "disabled" 😆